dashboard-widgets.el

```diff
@@ -174,6 +174,8 @@ Example:
         (setq package-count (length package-activated-list)))
       (when (boundp 'straight--profile-cache)
         (setq package-count (+ (hash-table-count straight--profile-cache) package-count)))
+	  (when (fboundp 'elpaca--queued)
+		(setq package-count (length (elpaca--queued))))
```

Comment:
Let's use spaces instead of tab here! And the indentation seems to be incorrect. Can you please fix that also? Thanks!

Reply:
Whoops, I ignored the project's .dir-locals.el, my bad. The indentation is correct AFAIK, its the same as the two cases above.